### DISCUSSION TITLE
Using a VPN

### DISCUSSION THREAD

**Discussion by scottrobertson** [NoahFetz/F1AppleTV#24] [Q&A]
I normally watch on my Google TV using the ExpressVPN app, however that does not seem to be a thing on AppleTV. I've ordered an Apple TV so I'm interested to see how this app can work for me. 

Has anyone used this app using a VPN with success?

**Comment by NoahFetz** [comment]
AppleTV doesn't have VPN support. You can only use dns vpns

**Reply by scottrobertson** [reply]
Yeah that's kinda what I meant, but forgot to say it. 

Does the dns vpn work for F1TV?

**Reply by NoahFetz** [reply]
I can't tell you I don't use a VPN... You have to try it out

**Reply by scottrobertson** [reply]
Thanks. Gotta wait until the 21st for my 4K Apple TV to arrive. 

I asked in Discussions as I mainly just wanted to ask the community. Not expecting you to have the answer, don't worry!
